src/common/console.ml

```diff
@@ -22,9 +22,10 @@ let set_default_verbose : int -> unit = fun i ->
     [lvl] is strictly greater than the current verbosity level.  Note that the
     output channel is automatically flushed if logging modes are enabled. *)
 let out : int -> 'a outfmt -> 'a = fun lvl fmt ->
-  if lvl > !verbose then Format.ifprintf Stdlib.(!out_fmt) fmt
-  else Format.fprintf Stdlib.(!out_fmt)
-         (if !log_enabled then fmt else fmt ^^ "%!")
+  if lvl > !verbose then
+    Format.ifprintf Stdlib.(!out_fmt) fmt
+  else
+    Format. fprintf Stdlib.(!out_fmt) fmt
```

Comment:
could be on 2 lines? and factorized?
`Format.(if lvl > !verbose then ifprintf else fprintf) Stdlib.(!out_fmt) fmt`